bundles/org.openhab.binding.lghombot/README.md

```diff
@@ -0,0 +1,95 @@
+# LG HomBot Binding
+
+The binding integrates hacked LG HomBot VR6260 based vacuum robots.
+Details on how to modify your HomBot can be [found here](https://www.roboter-forum.com/index.php?thread/10009-lg-hombot-3-0-wlan-kamera-steuerung-per-weboberfläche/).
+The binding uses the HTTP service on port 6260 right now.
+
+Please take care when modding your HomBot! This binding is a complement to a modded HomBot not an excuse to do the modding.
+Remember, You are responsible if You brick Your HomBot.
+
+## Supported Things
+
+Hacked LG HomBot series 62XX are supported.
+
+The service that the binding connects to is actually lg.srv [found here](https://sourceforge.net/projects/lgsrv/) running on a HomBot.
+
+## Discovery
+
+The auto-discovery should hopefully find your HomBot, be aware that it will try to connect to port 6260 on all IP-addresses on your subnet.
+If you already know the IP-address of your HomBot you might as well just plug it in.
+
+## Thing Configuration
+
+The thing only requires an IP-address to function, this should hopefully be found using the discovery method.
```

Comment:
I personally don't like (to express) this uncertainty it sounds like the binding is not very likely to work, I think it should be worded with more confidence.

Reply:
What is the name of the configuration parameter?